server/chat-commands.js

```diff
@@ -3748,6 +3748,30 @@ const commands = {
 		this.parse(`/help importinputlog`);
 	},
 
+	acceptdraw: 'offerdraw',
+	offerdraw(target, room, user) {
+		const battle = room.battle;
+		if (!battle) return this.errorReply("This command can only be used in battle rooms.");
+		if (!this.can('roomvoice', null, room)) return;
+		if (!target) {
```

Comment:
There shouldn't need to be a `target`. The offer to a draw should be sent to all other players in the room.

Reply:
I also think `X is offering a draw.` should be public. Only the "Accept draw", "Reject" buttons should be private.

Reply:
Only reason I used target was to be able to differentiate when it's `acceptdraw` or `offerdraw` but I guess I can use `cmd`?

Reply:
Oh wait, I can use `otherPlayer`!